.ci/main-trigger/Jenkinsfile

```
@@ -0,0 +1,50 @@
+// https://github.com/camunda/jenkins-global-shared-library
+// https://github.com/camunda/cambpm-jenkins-shared-library
+@Library(['camunda-ci', 'cambpm-jenkins-shared-library@3021']) _
+
+pipeline {
+  agent {
+    node {
+      label 'jenkins-job-runner'
+    }
+  }
+  environment {
+    LOGGER_LOG_LEVEL = 'DEBUG'
```

Do we need to specify the Maven version here as well?
```suggestion
    LOGGER_LOG_LEVEL = 'DEBUG'
    MAVEN_VERSION = 'maven-3.8-latest'
```

---

At the moment, it's not necessary as we don't run maven. However, I'll add it for consistency.